AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockImagePreviewSection.xml

```diff
@@ -8,7 +8,9 @@
 
 <sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
     <section name="AdobeStockImagePreviewSection">
-        <element name="save" type="block" selector="//button[@class='action-secondary']/span[contains(.,'Save Preview')]"/>
+        <element name="save" type="button" selector="//button[@class='action-secondary']//span[text()='Save Preview']"/>
+        <element name="locateImage" type="button" selector="//button[@class='action-secondary']//span[text()='Locate']"/>
+        <element name="saveLicensedImage" type="button" selector="//div[@class='actions']/descendant::span[text()='Save']"/>
```

Comment:
Hey @filmaj, sorry for the delay but I finally resolved the issue we encountered in this. 

-  Turns out that for some reason selenium isn't able to select the _Save_ button and clicks a totally different location (I'm thinking it might be due the fact that this button has multiple classes and is similar to _License and Save_, but I might be wrong). I fixed it by giving the _Save_ button a more specific selector and the test is successful now.

Reply:
Thanks! This part now passes for me locally.